src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc

```diff
@@ -22,12 +23,41 @@ namespace mirror {
 namespace image_replayer {
 namespace snapshot {
 
+using librbd::util::create_context_callback;
+
 template <typename I>
 void PrepareReplayRequest<I>::send() {
-  // TODO
   *m_resync_requested = false;
   *m_syncing = false;
 
+  load_local_image_meta();
+}
+
+template <typename I>
+void PrepareReplayRequest<I>::load_local_image_meta() {
+  dout(15) << dendl;
+
+  ceph_assert(m_state_builder->local_image_meta == nullptr);
+  m_state_builder->local_image_meta = new librbd::mirror::snapshot::ImageMeta(
+    m_state_builder->local_image_ctx, m_local_mirror_uuid);
```

Comment:
@dillaman shaman failed to build it [1]:
```
CMakeFiles/rbd_mirror_internal.dir/image_replayer/snapshot/PrepareReplayRequest.cc.o -c /build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc
/build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc: In member function 'void rbd::mirror::image_replayer::snapshot::PrepareReplayRequest<ImageCtxT>::load_local_image_meta()':
/build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc:42:58: error: class template argument deduction failed:
     m_state_builder->local_image_ctx, m_local_mirror_uuid);
                                                          ^
/build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc:42:58: error: no matching function for call to 'ImageMeta()'
In file included from /build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc:10:0:
/build/ceph-15.1.0-1216-g86045ba/src/librbd/mirror/snapshot/ImageMeta.h:27:3: note: candidate: template<class ImageCtxT> ImageMeta(ImageCtxT*, const string&)-> librbd::mirror::snapshot::ImageMeta<ImageCtxT>
   ImageMeta(ImageCtxT* image_ctx, const std::string& mirror_uuid);
   ^~~~~~~~~
/build/ceph-15.1.0-1216-g86045ba/src/librbd/mirror/snapshot/ImageMeta.h:27:3: note:   template argument deduction/substitution failed:
/build/ceph-15.1.0-1216-g86045ba/src/tools/rbd_mirror/image_replayer/snapshot/PrepareReplayRequest.cc:42:58: note:   candidate expects 2 arguments, 0 provided
     m_state_builder->local_image_ctx, m_local_mirror_uuid);
                                                          ^
src/tools/rbd_mirror/CMakeFiles/rbd_mirror_internal.dir/build.make:1241: recipe for target 'src/tools/rbd_mirror/CMakeFiles/rbd_mirror_internal.dir/image_replayer/snapshot/PrepareReplayRequest.cc.o' failed
```
[1] https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=bionic,DIST=bionic,MACHINE_SIZE=huge/37446//consoleFull